pkg/berglas/access.go

```diff
@@ -62,10 +65,13 @@ func (c *Client) Access(ctx context.Context, i *AccessRequest) ([]byte, error) {
 	}
 
 	// Get attributes to find the KMS key
-	attrs, err := c.storageClient.
+	h := c.storageClient.
 		Bucket(bucket).
-		Object(object).
-		Attrs(ctx)
+		Object(object)
+	if i.Generation != 0 {
+		h = h.Generation(i.Generation)
```

Comment:
I actually think you could just chain `Generation` onto the existing list, since that's the default, unset value for int64.

Reply:
unfortunately, `storage` uses `-1` as the [default value](https://github.com/maroux/berglas/blob/28454e13b14573c461492940df82c5129f40de53/vendor/cloud.google.com/go/storage/bucket.go#L149), instead of 0. Perhaps `0` _can_ be a valid GCS object version (not sure).